app/views/shop_account/shop_orders/index.html.haml

```diff
@@ -0,0 +1,16 @@
+%h1 Order Detail
+%table.table.table-bordered
+  %thead.thead-dark
+    %tr
+      %th Customer
+      %th Order_id
+      %th Order_date
+      %th Total
+      %th
+  %tbody
+    - if @orders.present?
```

Comment:
Not necessary, if there is no order, the each statement will not run

Reply:
oke em